### ISSUE TITLE
Question of the wiring between Arduino Mega and Ardumoto

### ISSUE THREAD

**Issue by yhuangece7** [cswiger/dcservodrive#2]
Hi cswiger,
     Thank you for providing such a good example of the servo motor design. I am currently using Arduino Mega and Ardumoto ([the latest version](https://learn.sparkfun.com/tutorials/ardumoto-kit-hookup-guide?_ga=2.188719016.1270687951.1677972540-1103759794.1677311178&_gac=1.81089253.1677311178.Cj0KCQiA3eGfBhCeARIsACpJNU_F3bYB09aoRRZx8R6IokcRfCkD3bl_AlyPv6YUmK9FvxUWVvx6SRkaApiZEALw_wcB#example-code)) to build something similar, but I have one question regarding the code and the corresponding wiring. Maybe my question is not related to the problem of your code, but is due to that I did not understand your code entirely. Therefore, any inputs are highly appreciated.
      My question is: I assume an external signal "STEP" and "DIR" comes into Arduino Mega, along with the feedback input from the encoders. When the Arduino Mega process the signals, it is expected to generate two output signals (say "STEP_PROCESSED" and "DIR_PROCESSED" to feed into Ardumoto. However, I did not find you have defined any GPIO pins in your code regarding that. 
      In summary, for the first motor,  I do not what are my "STEP_PROCESSED" and "DIR_PROCESSED" in Arduino Mega to connect to "IN1" and "PWM1" in Ardumoto. Would you advise what is my problem? 
     Thank you so much for your help!
